app/views/distributions/_pickup_day_row.html.erb

```diff
@@ -0,0 +1,17 @@
+<tr>
+  <td><%= pickup_day_row.partner.name %></td>
+  <td><%= pickup_day_row.issued_at.strftime("%l:%M %p") %> </td>
+  <td><%= pickup_day_row.storage_location.name %></td>
+  <td><%= pickup_day_row.line_items.total %></td>
+  <td>
+    <% if pickup_day_row.complete? %>
+      Picked up
+    <% else %>
+      Scheduled
+    <% end %>
+  </td>
+  <td class="text-right">
+    <%= update_button_to picked_up_distribution_path(pickup_day_row), {text: "Pick up Complete", size: "xs"} if pickup_day_row.scheduled? %>
```

Comment:
If we want to keep the 3 states, maybe we change this to `if !pickup_day_row.complete?`